### PR TITLE
Add grouping for Chewy journal entries

### PR DESCRIPTION
**Pull request by sergey-kintsel**

### CODE REVIEW

File: lib/chewy/journal.rb [hunk 1]
```diff
@@ -118,5 +120,21 @@ def query(time, comparator, query_type = :filter)
         }
       end
     end
+
+    Entry = Struct.new(:index_name, :type_name, :action, :object_ids, :created_at) do
```
Comment (pyromaniac) [inline lib/chewy/journal.rb]: JournalEntry and maybe makes sense to extract if to the own file

Comment (sergey-kintsel) [inline lib/chewy/journal.rb]: No™

Comment (pyromaniac) [inline lib/chewy/journal.rb]: You will regret about this.

Comment (sergey-kintsel) [inline lib/chewy/journal.rb]: Maybe, but who knows what will be 😄

Comment (pyromaniac) [inline lib/chewy/journal.rb]: I know

File: lib/chewy/journal.rb [hunk 2]
```diff
@@ -118,5 +120,31 @@ def query(time, comparator, query_type = :filter)
...
       end
     end
+
+    class Entry
+      ATTRIBUTES = %w[index_name type_name action object_ids created_at].freeze
+
+      attr_accessor *ATTRIBUTES
+
+      def initialize(attributes = {})
+        attributes.slice(*ATTRIBUTES).each do |attr, value|
```
Comment (pyromaniac) [inline lib/chewy/journal.rb]: attributes.stringify_keys.slice

**Pull request merged**